README.md

```diff
@@ -80,11 +80,13 @@ Usage:
   sensu-influxdb-handler [flags]
 
 Flags:
-  -a, --addr string       the address of the influxdb server, should be of the form 'http://host:port'
-  -d, --db-name string    the influxdb to send metrics to
-  -h, --help              help for sensu-influxdb-handler
-  -p, --password string   the password for the given db
-  -u, --username string   the username for the given db
+  -a, --addr string            the address of the influxdb server, should be of the form 'http://host:port'
+  -d, --db-name string         the influxdb to send metrics to
+  -h, --help                   help for sensu-influxdb-handler
+  -i, --insecure-skip-verify   if true, the influx client skips https certification validation
```

Comment:
I would personally phrase that as just "skip TLS certificate validation".

Reply:
@ccressent This value is derived from the influx client, so I just used their docs: https://github.com/influxdata/influxdb/blob/367328aa3eb74c86674a922eab54434ccda3d570/client/v2/client.go#L43-L45

I can certainly change the help usage if you'd prefer!

Reply:
Thanks for the reference; good idea to keep it the same as Influx's documentation. My only gripe is that it should be "certificate" and not "certification", just like in Influx!

Reply:
You're totally right, good catch!